backend/z_index.js

```diff
@@ -36,6 +37,26 @@ app.get('/absences', async (req, res) => {
   const absences = await getAbsenceForms();
   res.json(absences);
 })
+app.get('/meetingHistory', async (req,res) => {
+  const meetingHist = await getMeetingHistory();
+  res.json(meetingHist);
+})
+app.post('/updateDatabases', (req, res) => {
+  if (req.body == undefined) {
+    return res.json({
+      msg: "Error: parameter not defined",
+      data: {}
```

Comment:
Try to return a status code whenever possible since I beleive that this still sends back status code 200

https://stackoverflow.com/questions/54614705/nodejs-express-how-to-return-http-error-code-400-401-500-plus-json-object